UnityProject/Assets/Scripts/Items/Botany/Fruit/NoFruit.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace Items.Botany.Fruit
+{
+	public class NoFruit : NetworkBehaviour, ICheckedInteractable<HandApply>
+	{
+
+		[SyncVar] private bool isReadyToBeHit = false;
+		private SpriteHandler handler;
+
+		[SerializeField] private List<GameObject> chanceToSpawn = new List<GameObject>();
+		[SerializeField] private GameObject facehugger;
+		//Since we can't get the sprites of prefabs before spawning them, we'll have to use a pre-made list that is made from the inspector
+		[SerializeField] private List<SpriteDataSO> spritesToShow = new List<SpriteDataSO>();
```

Comment:
you can't iterate ``chanceToSpawn`` and get component from them?

Reply:
As stated on discord, Unity doesn't like it when you try accessing data from gameobjects that are not in memory yet. You have to spawn them, get the data, then free them afterwards.